src/_alfasim_sdk/alfacase/generate_case_description_docstring.py

```diff
@@ -83,7 +111,21 @@ def _get_declaration(
             for predicate_function, handle_function in list_of_predicate_and_handles:
                 if predicate_function(value.type):
                     attribute_value = handle_function(value.type)
-                    lines.append(f"{BASE_INDENT}{attribute_name}: {attribute_value}")
+                    default_value = _get_default_value_of_case(value)
+
+                    # For CaseDescription, if the attributes has a default value, the empty space is not necessary
+                    if default_value:
+                        attribute_value = attribute_value.replace("|space|", "")
+
+                    # For Schema,  if the attributes has a default value, the entry is optional
+                    if default_value and is_schema:
+                        attribute_value = attribute_value.replace("# optional", "")
+                        attribute_value += " # optional"
```

Comment:
If your intent is just to add an extra space before `# optional` you could just:
>  attribute_value = attribute_value.replace("# optional", " # optional")

Reply:
Nops, I'm adding `#optional` to the fields that have a default value.

Previously this `optional` label was being added only on fields that were `Optional[Something]`.
Perhaps I can change that since all options that are `Optional[Something]` have a default value.

Reply:
Done on 22af421